projects/demo/src/modules/experimental/chip/examples/4/index.html

```diff
@@ -0,0 +1,47 @@
+<tui-chip
+    appearance="primary"
+    class="fade"
+>
+    <div tuiFade>Very long value in chip</div>
+    <div>{{ 123435 | tuiAmount: 'RUB' | async }}</div>
+</tui-chip>
+
+<tui-chip
+    appearance="accent"
+    iconLeft="tuiIconBox"
+>
+    <div class="ellipsis">Very long value in chip</div>
+</tui-chip>
+
+<tui-badged-content>
+    <tui-badge-notification
+        size="xs"
+        tuiSlot="top"
+    ></tui-badge-notification>
+    <tui-chip
+        appearance="secondary"
+        iconLeft="tuiIconBell"
+    >
+        Notifications
```

Comment:
**Chrome:**
<img height="300" alt="chrome" src="https://github.com/taiga-family/taiga-ui/assets/35179038/411d1c46-3e85-4b3c-b043-3397d40d34e5">

___

**Safari:**
<img height="300" alt="safari" src="https://github.com/taiga-family/taiga-ui/assets/35179038/3e3f06d3-ca6d-43f2-8a61-de0c259dc96d">

Reply:
Less gap between `Bell`-icon and word "Notifications" 🤔